src/docfx/Errors.cs

```diff
@@ -338,6 +338,27 @@ public static Error CircularRedirection(SourceInfo? source, IEnumerable<FilePath
             /// Behavior: ✔️ Message: ✔️
             public static Error RedirectedFileNotRemoved(FilePath path)
                 => new Error(ErrorLevel.Warning, "redirected-file-not-removed", $"Redirected file '{path}' is still in the repo. After adding a file to the redirection JSON file, you must delete the original file from the repo.");
+
+            /// <summary>
+            /// A redirection file registered in .openpublishing.publish.json is not found in the repo.
+            /// </summary>
+            /// Behavior: ✔️ Message: ✔️
+            public static Error FileNotFound(string path)
+                => new Error(ErrorLevel.Error, "file-not-found", $"Redirection file '{path}' registered in .openpublishing.publish.json is not found in the repo.");
+
+            /// <summary>
+            /// A redirection item cannot contain ‘source_path’ and ‘source_path_from_root’ at the same time.
+            /// </summary>
+            /// /// Behavior: ✔️ Message: ✔️
+            public static Error SourcePathConflict(SourceInfo<string> source)
+                => new Error(ErrorLevel.Error, "source-path-conflict", $"A redirection item cannot contain 'source_path' and 'source_path_from_root' at the same time.", source);
```

Comment:
It would be better to specify which one is in use in the message.

Reply:
No one will be used when they exist at the same time.

All error codes and messages are confirmed with PM. I think we shouldn't change them.